packages/tracing/src/idletransaction.ts

```diff
@@ -69,25 +72,27 @@ export class IdleTransaction extends Transaction {
   private readonly _beforeFinishCallbacks: BeforeFinishCallback[] = [];
 
   /**
-   * If a transaction is created and no activities are added, we want to make sure that
-   * it times out properly. This is cleared and not used when activities are added.
+   * Timer that tracks a
    */
-  private _initTimeout: ReturnType<typeof setTimeout> | undefined;
+  private _idleTimeoutID: ReturnType<typeof global.setTimeout> | undefined;
 
   public constructor(
     transactionContext: TransactionContext,
-    private readonly _idleHub?: Hub,
+    private readonly _idleHub: Hub,
     /**
      * The time to wait in ms until the idle transaction will be finished.
```

Comment:
Doesn't have to be worded this way but it took me a while to understand when this timer should be started and stopped.

```suggestion
     * The time to wait in ms until the idle transaction will be finished. This timer is started each time there are no active spans on this transaction.
```